lib/AST/FineGrainedDependencyFormat.cpp

```diff
@@ -44,7 +44,7 @@ class Deserializer {
 
 public:
   Deserializer(llvm::MemoryBufferRef Data) : Cursor(Data) {}
-  bool readFineGrainedDependencyGraph(SourceFileDepGraph &g);
+  bool readFineGrainedDependencyGraph(SourceFileDepGraph &g, bool forSwiftDeps);
```

Comment:
It feels like there is an intension shift missing from the code:
"forSwiftDeps vs forSwiftModule" -> "includeWrapping vs excludeWrapping" or some such.
The comment in the .h sorta says it, but I wonder if a different name? Even an extra layer, though it would be so short that some I know would not like it.